server/src/main/java/no/ntnu/klubbhuset/service/UserService.java

```diff
@@ -67,9 +73,12 @@ public Response createNewUser(User user) {
         if (user == null) {
             return Response.status(Response.Status.FORBIDDEN).build();
         }
+        SecurityGroup securityGroup = entityManager.find(SecurityGroup.class, SecurityGroup.USER);
+        user.addSecurityGroup(securityGroup);
+        user.setPassword(hasher.generate(user.getPassword().toCharArray()));
         entityManager.persist(user);
 
-        return Response.status(Response.Status.CREATED).entity(user).build();
+        return Response.status(Response.Status.CREATED).build();
```

Comment:
Shouldn't we send some data back to the client? If not the user maybe a token. So that they are logged in as soon as they create a user?

Reply:
That is something we definetely can do, but the whole idea of a token is still a bit unclear to me. I think we should ask Mikael for an explanation on how Tokens work, and how we should continue using tokens in our system.

I don't think we should include anytthing else in this pull request, but instead just create another issue on the whole token situation, and a more comfortable login flow.

Reply:
Agreed. Let's do it